config/web.php

```diff
@@ -9,9 +9,6 @@
     'components' => [
         'db' => [
             'class'     => 'yii\db\Connection',
-            'dsn'       => 'mysql:host=127.0.0.1;dbname=walle',
-            'username'  => 'root',
```

Comment:
The changes of `config/*` should not be submit, but you can keep it local. Because something such as db is need by travis which is a integration tool, I would reset it later : )